src/middleware/logRequest.js

```diff
@@ -0,0 +1,17 @@
+module.exports.logRequest = async (event, context) => {
```

Comment:
You created a logger for lambdas but you don't use it at all. How come?

Reply:
I do use it, added in serverless.yml file as middleware. It is not visible in PR as serverless.yml is in gitignore because of db credentials. I added a comment in the PR in the Optional tasks section, and screenshots illusrating that logs really happen. Please have a look :)

Reply:
Sorry, but how can I see it if you don't include YML file into PR? Please be more prepared for the next review and do PRs in the correct.

You can just don't put credentials into file but you have to put YML file into your repository

Reply:
As far as I understood, in this task we should have excluded serverless.yml from repo (and keep credentials there for db).
I added screenshots of lambda logs for each endpoint (see PR), would this be sufficient?
Let me also add a screen of how the logger is added in serverless.yml (sorry, have no idea how to keep db settings data in serverless.yml and still show the logger work which is happening there at the same time): 
https://i.imgur.com/LICjXjA.png

Reply:
Your AWS services are not connected with your repository. You can put credentials, deploy service, remove credentials, create MR without them.

Reply:
Ah, you mean just remove manually! It's possible, though very tedious) let me do this once :)

Reply:
Done, please take a look)